tests/unit_tests/db/test_db_handler.py

```diff
@@ -2,16 +2,33 @@
 
 import copy
 import logging
-import re
 import uuid
+from pathlib import Path
+from unittest.mock import call
 
 import pytest
+from bson.objectid import ObjectId
 
 from simtools.db import db_handler
+from simtools.utils import names
 
 logger = logging.getLogger()
 
 
+@pytest.fixture(autouse=True)
+def reset_db_client():
```

Comment:
How do you make sure that this is called after the `_db_cleanup` fixture? If it's called before, then the sanboxes will not be deleted (or perhaps an error would occur).

Reply:
Very good point. I have been wondering where all the sandboxes come from in the DB (just removed 128 sandboxes). I've combined both functions now into one to make sure that DBs are removed in the correct sequence. Good catch!

Reply:
Excellent